rocket/workspace/hello-world/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "hello-world"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+rocket = "0.5.0-rc.2"
+shared = { path = "../shared", version = "0.1.0" }
+shuttle-rocket = { version = "0.13.0" }
+shuttle-runtime = { version = "0.13.0" }
```

This should be committed before merging this for the release.
```suggestion
shuttle-rocket = { version = "0.14.0" }
shuttle-runtime = { version = "0.14.0" }
```